weather/module.py

```diff
@@ -98,9 +188,25 @@ async def weather(self, ctx, name: Optional[str] = None):
             await ctx.reply(_(ctx, "You have to specify a place or set a preference."))
             return
 
-        await ctx.reply(name)
-
-    #
+        # try to get user language preference
+        lang_preference = MemberLanguage.get(guild_id=ctx.guild.id, member_id=ctx.author.id)
+        if lang_preference is not None:
+            lang_preference = getattr(lang_preference, "language", None)
+    
+        if lang_preference is None:
+            # try to get guild language preference
+            lang_preference = GuildLanguage.get(guild_id=ctx.guild.id)
+            if lang_preference is not None:
+                lang_preference = getattr(lang_preference, "language", None)
+            else:
+                lang_preference = 'en'
+
+        embeds = self._create_embeds(ctx, name, lang_preference)
+        if not isinstance(embeds, list):
```

Comment:
I think that _create_embeds should always return list of embeds, even if it's only one embed. Then you can pass that list to ScrollableEmbed, which can handle single item list by sending it as is without scrolling option.